notebooks/ir/dense/dense_ir_pipeline.ipynb

```diff
@@ -584,7 +584,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.13"
+   "version": "3.9.15 (main, Nov 24 2022, 14:31:59) \n[GCC 11.2.0]"
+  },
+  "vscode": {
```

Comment:
I'm not a vscode user - is this hash necessary to be kept?

Reply:
I think this gets automatically generated when you run the notebook in vscode. It will keep on changing its a sort of meta data attached to ipynb.